instrumentation/oshi/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/oshi/AbstractOshiInstrumentation.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.oshi;
+
+import io.opentelemetry.javaagent.tooling.Instrumenter;
+
+public abstract class AbstractOshiInstrumentation extends Instrumenter.Default {
```

Comment:
since only one Instrumentation class, don't need to break out abstract base class

Reply:
removed